embedded/store/immustore_test.go

```diff
@@ -306,13 +306,13 @@ func TestImmudbStoreEdgeCases(t *testing.T) {
 		require.EqualError(t, err, "stat invalid\x00_dir_name: invalid argument")
 	})
 
-	t.Run("should fail with permiission denied", func(t *testing.T) {
```

Comment:
is this test case failing in the github runner? Recent changes shouldn't affect this edge case... However I was able to reproduce it locally when running test inside a container but with macOS as host

Reply:
this test is only failing on a mac, and also in ubuntu container. I will uncomment this and run in gitlab runner